### ISSUE TITLE
Support for Helm v3.0.0 and Chart version (v2)

### ISSUE THREAD

**Issue by vamsikurre** [presslabs/mysql-operator#433]
Hi, 
New Helm v3.0.0 stable was released and looks like there are backward compatibility breakages,
[https://github.com/helm/helm/releases/tag/v3.0.0](url)

Is there plan for supporting Chart version (v2)?

Getting below error when I tried to install operator with helm3, had  to install helm2 

```
$ helm install mysql-operator presslabs/mysql-operator --namespace orbit1
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
```

Thanks

**Comment by AMecea** [comment]
Hi @vamsikurre, 

We will like to add support for helm version 3.  We can make the chart compatible with both v3 and v2. 

If you are willing to contribute to this, PR are welcomed.

**Comment by 21h** [comment]
https://github.com/presslabs/mysql-operator/pull/441

**Comment by AMecea** [comment]
Addressed by #468 and will be available in version `0.4.0`
